### ISSUE TITLE
Add power off delay function

### ISSUE THREAD

**Issue by dj5927** [southoz/RetroOZ#27]
It would be nice to be able to set a delay on the power button.
I wish there was an option to shut it down by holding it down for 2-3 seconds.
Too many situations where you accidentally click and exit.

**Comment by JuanMiguelBG** [comment]
Yes, it's a great suggestion. But all solutions I see need ACPI.

 [Double pressed solution](https://github.com/stackcoder/doublepress)

**Comment by southoz** [comment]
The power key handling is based on /dev/input/event0 which I am fairly sure is handled by something watching /dev/input/by-path/platform-ff180000.i2c-event to issue the shutdown command.
